modules/pilotxBidAdapter.js

```diff
@@ -0,0 +1,147 @@
+          }
+        } else {
+          sizes[0] = [bidRequest.sizes[0], bidRequest.sizes[1]]
+        }
+        payloadItems[placementId]['sizes'] = sizes
+      }
+      if (bidRequest.mediaTypes != null) {
+        for (let i in bidRequest.mediaTypes) {
+          payloadItems[placementId][i] = {
+            ...bidRequest.mediaTypes[i]
+          }
+        }
+      }
+      let consentTemp = ''
+      let consentRequiredTemp = false
+      if (bidderRequest && bidderRequest.gdprConsent) {
+        consentTemp = bidderRequest.gdprConsent.consentString
+        // will check if the gdprApplies field was populated with a boolean value (ie from page config).  If it's undefined, then default to true
+        consentRequiredTemp = (typeof bidderRequest.gdprConsent.gdprApplies === 'boolean') ? bidderRequest.gdprConsent.gdprApplies : true
+      }
+
+      payloadItems[placementId]['gdprConsentString'] = consentTemp
+      payloadItems[placementId]['gdprConsentRequired'] = consentRequiredTemp
+      payloadItems[placementId]['bidId'] = bidRequest.bidId
+    });
+    const payload = payloadItems;
+    const payloadString = JSON.stringify(payload);
+    return {
+      method: 'POST',
+      url: ENDPOINT_URL,
+      data: payloadString,
+    };
+  },
+  /**
+         * Unpack the response from the server into a list of bids.
+         *
+         * @param {ServerResponse} serverResponse A successful response from the server.
+         * @return {Bid[]} An array of bids which were nested inside the server.
+         */
+  interpretResponse: function (serverResponse, bidRequest) {
+    const serverBody = serverResponse.body;
+    const bidResponses = [];
+    if (serverBody.mediaType == 'banner') {
+      const bidResponse = {
+        requestId: serverBody.requestId,
+        cpm: serverBody.cpm,
+        width: serverBody.width,
+        height: serverBody.height,
+        creativeId: serverBody.creativeId,
+        currency: serverBody.currency,
+        netRevenue: false,
+        ttl: serverBody.ttl,
+        ad: serverBody.ad,
+        mediaType: 'banner',
+        meta: {
+          mediaType: 'banner'
+        }
+      }
+      bidResponses.push(bidResponse)
+    } else if (serverBody.mediaType == 'video') {
+      const bidResponse = {
+        requestId: serverBody.requestId,
+        cpm: serverBody.cpm,
+        width: serverBody.width,
+        height: serverBody.height,
+        creativeId: serverBody.creativeId,
+        currency: serverBody.currency,
+        netRevenue: false,
+        ttl: serverBody.ttl,
+        vastUrl: serverBody.vastUrl,
+        mediaType: 'video',
+        meta: {
+          mediaType: 'video'
+        }
+      }
+      bidResponses.push(bidResponse)
+    }
+
+    return bidResponses;
+  },
+
+  /**
+   * Formats placement ids for adserver ingestion purposes
+   * @param {string[]} The placement ID/s in an array
+   */
+  setPlacementID: function (placementId) {
+    if (Array.isArray) {

```

Comment:
no need for this check, Array.isArray is supported on all browsers PBJS supports, you can just use it as you did on L50